lilliput.go

```diff
@@ -77,7 +78,10 @@ func isGIF(maybeGIF []byte) bool {
 }
 
 func isWebp(maybeWebp []byte) bool {
-	return bytes.HasPrefix(maybeWebp, webpMagic)
+	if len(maybeWebp) < 12 {
```

Comment:
im guessing this is just a safety check to ensure we don't get an index out of bounds error?